base/etc/osg/image-config.d/20-osg-ce-setup.sh

```diff
@@ -63,6 +63,14 @@ echo ">>>>> YOUR CERTIFICATE INFORMATION IS:"
 openssl x509 -in $hostcert_path -text
 echo "><><><><><><><><><><><><><><><><><><><"
 
-chown -R condor:condor /var/log/condor-ce /var/lib/condor-ce/spool
+# Ensure that PVC dirs and subdirs exist and have the proper
+# permissions (SOFTWARE-4423)
```

Comment:
real minor, but since you are only setting the owner and not touching the permission (mode) bits,

```suggestion
# ownership (SOFTWARE-4423)
```

Reply:
Oh duh, thanks